pkg/master/tunneler/ssh.go

```diff
@@ -175,7 +172,7 @@ func (c *SSHTunneler) installSSHKeySyncLoop(user, publicKeyfile string) {
 			glog.Errorf("Failed to encode public key: %v", err)
 			return
 		}
-		if err := c.InstallSSHKey(user, keyData); err != nil {
+		if err := c.InstallSSHKey(user, keyData); err != nil && err != cloudprovider.NotImplemented {
```

Comment:
@sttts @liggitt 
Any other ideas? 
Implementing a function on each cloud-provider, which tells it this functionality is implemented, seems a bit overkill and not pretty either.